### PR TITLE
Newsweek: request project approval

### PR DESCRIPTION
**Pull request by Gnwin**
## Newsweek Clone completion.

In this project, we created the Newsweek website at www.newsweek.com using:
The Bootstrap framework,
HTML,
and CSS.

### CODE REVIEW

File: assets/stylesheets/style.css [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+}
+
+/* Mobile View */
+
+.header1 {
+	background-color: #f72210;
+	padding: 0.8rem;
+	/* position: fixed;
+	width: 100%;
+	z-index: 2; */
```
Comment (PhillipUg) [inline assets/stylesheets/style.css]: - [x] Remove all commented pieces of code

**Review by PhillipUg — CHANGES_REQUESTED** [review]
Hey **Ana Dessoy Hübner & Godwin Nwachukwu**, 👋

You have done a great job on this project!  

However, you need to work on a couple of project requirements before you can move on to the next project

### Changes Required ♻️ 

- [x] You have some linter errors. Please make sure you don't have any linter errors before you submit for review

- [x] The hamburger icon that appears at the extreme end on the right should not be present on the large screen size unless you scroll down or if you're on the tablet and mobile screen sizes.

- [x] The sports section, towards the bottom has only one item on the right, unlike the others which have three. Please check it out

- [x] On mobile screen size, padding appears towards the right. please fix this

Original Page | Your page
------------ | -------------
www newsweek com_  | gnwin github io_Godwin-Ana-Hubner-Newsweek-project_

### Optional Changes:

If you want to make your project even better 🔥💥, I left a few suggestions you can implement. 
These are completely [Optional]: 🦔

- [x] On mobile and tablet screen sizes, your navbar hides halfway every time I scroll down

- [ ] You excluded the tags on some images like `Tech & Science` and `News`. see image below
screenshot-www newsweek com-2020 11 07-10_53_24

- [x] On large screen, your navbar is not fixed. it disappears when you scroll down

#### Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack as **Phillip Musiime**
I would be happy to help! 👩‍💻 


Happy coding 🤓

**Comment by Gnwin** [comment]
## Newsweek Clone Completion

All required changes have been applied

**Review by PhillipUg — APPROVED** [review]
# Status: Project Approved! 🥇 

Congratulations on making all the requested changes and getting this project approved. :+1:

- [OPTIONAL]: Consider adding screenshots to your README to make it look more professional

You can merge this branch into the master branch when you're ready and go on to your next project.
:bulb:  Do not forget to submit your project completion form after merging this branch.

All the best team,

For any assistance, you can contact me on slack as: **Phillip Musiime**

**Pull request merged**